packages/combobox/stories/combobox-sizes.stories.ts

```diff
@@ -34,14 +35,14 @@ const combobox = ({
     size: 's' | 'm' | 'l' | 'xl';
 }): TemplateResult => {
     const options: ComboboxOption[] = [
-        { id: 'thing1', value: 'Abc Thing 1' },
-        { id: 'thing1a', value: 'Bde Thing 2' },
-        { id: 'thing1b', value: 'Bef Thing 3' },
-        { id: 'thing4', value: 'Efg Thing 4' },
-        { id: 'athing1', value: 'Abc Thing 1' },
-        { id: 'athing1a', value: 'Bde Thing 2' },
-        { id: 'athing1b', value: 'Bef Thing 3' },
-        { id: 'athing4', value: 'Efg Thing 4' },
+        { value: 'thing1', itemText: 'Abc Thing 1' },
+        { value: 'thing1a', itemText: 'Bde Thing 2' },
+        { value: 'thing1b', itemText: 'Bef Thing 3' },
+        { value: 'thing4', itemText: 'Efg Thing 4' },
+        { value: 'athing1', itemText: 'Abc Thing 1' },
+        { value: 'athing1a', itemText: 'Bde Thing 2' },
+        { value: 'athing1b', itemText: 'Bef Thing 3' },
+        { value: 'athing4', itemText: 'Efg Thing 4' },
     ];
```

Comment:
Can we use one of the arrays from `stories/index.ts` here? 😅 I forgot to change that for these stories....

Reply:
Handled!